docs/data-scientists/experiment-sizing.md

```diff
@@ -0,0 +1,65 @@
+---
+id: experiment-sizing
+title: Sizing Experiments Using Mozanalysis
+slug: /experiment-sizing
+---
+
+This page gives an overview of how Mozanalysis can be used to do experiment sizing. **[Mozanalysis]** is a library used to standardize experiment analysis at Mozilla. Mozanalysis contains the statistical tools used by [Jetstream] to produce results from experiments, as well as tools to pull historical data from BigQuery to calculated necessary sample sizes to carry out experiments.
+
+## Experiment sizing paradigm
+
+Experiment sizing in Mozanalysis retrieves historical data from BigQuery in a way that mimicks how experiment enrollment and analysis is conducted by [Jetstream]: an enrollment period is defined, during which clients that satisfy certain target conditions are recorded; following enrollment, metrics are calculated for those clients over a defined analysis period. Following collection of metrics, sample size calculation is performed that corresponds to the statistical tests to be used in analyzing the eventual experiment. 
+
+<img src="/img/mozanalysis/mozanalysis-sizing.png" alt="Mozanalysis sizing workflow" className="img-lg"/>
+
+### Enrollment and analysis
+
+Similar to how experiments use an enrollment period to select clients for a study before collecting metrics for a period of time, Mozanalysis experiment sizing records all clients that satisfy the target conditions during an enrollment period, set by the user based on the analysis start date and the number of dates to enroll. Metrics are then recorded for a user-defined number of days for the analysis.
+
+<img src="/img/mozanalysis/enrollment-period.png" alt="Analysis with an enrollment period" className="img-lg"/>
+
+An alternative to using an enrollment period to select clients is to use continuous enrollment, where any client that satisfies the targeting criteria during the entire analysis period is enrolled; metrics are collected from each client's enrollment date to the end of the analysis period.
+
+<img src="/img/mozanalysis/continuous-enrollment.png" alt="Analysis with continuous enrollment" className="img-lg"/>
```

Comment:
What's the purpose of an enrollment window under a continuously enrolling experiment? Why do we "end enrollment at start date + analysis length" but also enrollment isn't ended?

Reply:
Basically to set an end date for the analysis, though I see now that this isn't fulfilling the purpose I had in my head: what it's doing now is just using an enrollment period equal to the collection period from the non-continuous enrollment. I'm going to leave it as-is in the documentation, but I'll add to my list making dates for a continuous enrollment analysis perform how we'd expect.

Reply:
Maybe I'm not understanding. How is the experiment end date related to enrollment end date here? Aren't those just separate things?

Reply:
In typical experiments, yeah, they're separate (experiment end is enrollment length + analysis length). In the case of continuous enrollment, experiment end is (first enrollment date + analysis length) + (last enrollment date + analysis length). So the code should be refactored to create a separate study duration and analysis length